src/wallet/rpc/wallet.cpp

```diff
@@ -59,6 +59,7 @@ static RPCHelpMan getwalletinfo()
                         }, /*skip_type_check=*/true},
                         {RPCResult::Type::BOOL, "descriptors", "whether this wallet uses descriptors for scriptPubKey management"},
                         {RPCResult::Type::BOOL, "external_signer", "whether this wallet is configured to use an external signer such as a hardware wallet"},
+                        {RPCResult::Type::BOOL, "has_legacy", "whether this wallet contain a legacy scriptPubKey manager or not"},
```

Comment:
I can see how it's useful in a test, but otherwise it's too much of an implementation detail, and potentially confusing.

Reply:
I don't think that differs much from the returned 'descriptors' flag. The information might be useful for debugging users' issues.
But.. I'm not so strong for it neither. Flag removed.

Reply:
It sounds like almost a mix between `descriptors` and `!blank`, but whether or not a legacy wallet contains a (in memory) `LegacySPKM` should not matter to the user. I didn't even know we don't create an empty one in all cases.